src/main/java/com/ibm/cloud/sdk/core/http/InputStreamRequestBody.java

```diff
@@ -60,6 +62,12 @@ private InputStreamRequestBody(InputStream inputStream, MediaType mediaType) {
     }
 
     this.bytes = outputStream.toByteArray();
+    try {
+      outputStream.close();
+    } catch (IOException e) {
+      LOGGER.severe("Could not close inputStream byte array.");
```

Comment:
```suggestion
      LOGGER.log(Level.SEVERE, "Could not close inputStream byte array.", e);
```
then remove the e.printStackTrace() call.
This way, the stacktrace will be part of the actual error message and would end up in the same place as other log messages instead of being sent to stderr.

Reply:
Good call, thanks 👍 

Also that fancy new suggestion feature is pretty fun 😮